packages/validator/test/session.spec.js

```diff
@@ -310,3 +310,47 @@ test('resolve key', async () => {
     },
   })
 })
+
+test('service can not delegate access to account', async () => {
+  const account = Absentee.from({ id: 'did:mailto:web.mail:alice' })
```

Comment:
the other test didn't use `Absentee` and it doesn't seem used beyond `.did()` (just noting things that might make it different from other test)